src/test/java/network/radicle/jetbrains/radiclejetbrainsplugin/patches/RadicleToolWindowTest.java

```diff
@@ -28,26 +31,22 @@ public class RadicleToolWindowTest extends AbstractIT {
     private static final String AUTHOR = "did:key:testAuthor";
     private RadicleToolWindow radicleToolWindow;
 
+    private List<RadPatch> getTestPatches() {
+        var author = new RadPatch.Author(AUTHOR);
+        var revision = new RadPatch.Revision("testRevision", "testDescription", "", "",
+                List.of(), List.of(), Instant.now(), List.of(), List.of());
+        var radPatch = new RadPatch("c5df12", "testPatch", author, "testDesc", "testTarget",
+                List.of("tag1", "tag2"), RadPatch.State.OPEN, List.of(revision));
+        return List.of(radPatch);
+    }
+
     @Before
     public void setUpToolWindow() throws InterruptedException, IOException {
         HttpClient httpClient = mock(HttpClient.class);
         HttpResponse httpResponse = mock(HttpResponse.class);
         StatusLine statusLine = mock(StatusLine.class);
-        StringEntity se = new StringEntity("[{\"id\":\"c5df12\",\"author\":{\"id\":\"" + AUTHOR + "\"},\"title\":\"My new project\"," +
-                "\"description\":\"Hello this is a description\",\"state\":{\"status\":\"open\"}," +
-                "\"target\":\"delegates\",\"tags\":[],\"revisions\":[{\"id\":\"testRevision\",\"description\":\"\"," +
-                "\"base\":\"9de952f92e27b67b43d89230e2d68caa0794b5a9\",\"oid\":\"b2a64eb1f42fd757a5ad03dfc5e13727ad52cb6e\"," +
-                "\"refs\":[],\"merges\":[],\"discussions\":[],\"timestamp\":1680864771,\"reviews\":[]}," +
-                "{\"id\":\"14e35ef75f6c7b73f965004f6f985934ba13bcd9\",\"description\":\"Add README, just for the fun\",\"base\":" +
-                "\"9de952f92e27b67b43d89230e2d68caa0794b5a9\",\"oid\":\"84fb6549088427b958903eafa52b6559b20756de\"," +
-                "\"refs\":[\"refs/heads/new_branch\"],\"merges\":[],\"discussions\":[{\"id\":\"c37274314cbe28efdf7c33abcaa01b3473797985\",\"author\":" +
-                "{\"id\":\"did:key:z6MkwM5cMjgsvTcUyEtUAfo1bWzGS9o41zXRcy7eqaZubwzS\"}," +
-                "\"body\":\"TestMsg!\",\"reactions\":[],\"timestamp\":1680867331,\"replyTo\":null},{\"id\":\"538fe004d02e84e554c3b49b70567bdd59cd48c5\"," +
-                "\"author\":{\"id\":\"did:key:z6MkwM5cMjgsvTcUyEtUAfo1bWzGS9o41zXRcy7eqaZubwzS\"}," +
-                "\"body\":\"This is my Reply\",\"reactions\":[],\"timestamp\":1680867502,\"replyTo\":\"c37274314cbe28efdf7c33abcaa01b3473797985\"}]," +
-                "\"timestamp\":1680865493,\"reviews\":[]}]}]");
+        StringEntity se = new StringEntity(getTestPatches().toString());
```

Comment:
similarly, here, it's better to use jackson to serialize the value, something like:
`new ObjectMapper().findAndRegisterModules().writeValueAsString(getTestPatches());`